### ISSUE TITLE
Shipping not following instructions 

### ISSUE THREAD

**Issue by mcastano24** [nbrightproject/NBrightBuy#151]
Hello, I am having an issue where the shipping is not following instructions, I have two shipping options, one regular and other express. for the purpose of this I put that  regular cost $10 and express costs $20. when there is only one item in the cart this works perfectly however as soon as you have more than one item even if it is the same item the shipping goes crazy. in this case express is still showing the correct amount of $20 however the regular is showing $13. and I have no idea where and why it is adding those $3

![1](https://user-images.githubusercontent.com/34758219/34378647-f4c099ea-eac4-11e7-8ab6-17609fde22f3.JPG)
![2](https://user-images.githubusercontent.com/34758219/34378648-f4cf6bbe-eac4-11e7-97b0-2e5f11fff453.PNG)
![3](https://user-images.githubusercontent.com/34758219/34378649-f4da6028-eac4-11e7-98b7-4e50fb8257df.PNG)
![4](https://user-images.githubusercontent.com/34758219/34378650-f4e5affa-eac4-11e7-964d-37b6ae57f4e8.PNG)

**Comment by leedavi** [comment]
Yes, this looks odd.  The calc is a simple calc to check the range and pass back the value, so that $3 is very strange.  Is it possible to send me a link and login to the site so I can look at it.

**Comment by mcastano24** [comment]
sure,i'll give you admin control when you register. we can delete it afterwards if you'd like

**Comment by leedavi** [comment]
OK, I registered. In future it's better to send a PM via: https://forum.nbrightproject.org/

**Comment by mcastano24** [comment]
yeah wasn't sure how to do that, anyway i know for next time. your profile is good to go

**Comment by leedavi** [comment]
You have "By Weight" selected, but no weight on the products.  Try adding a weight to the products and retry.

**Comment by mcastano24** [comment]
some of the items in the store have free shipping so I set the weight of them to "0" and then on the shipping range I made sure to use 1+ expect for express shipping. would that still be the issue if so how would i get around that

**Comment by leedavi** [comment]
OK, I just tested (without changing anything) and it worked OK??   Do another test and see if you can recreate the problem.
